braindecode/datasets/base.py

```diff
@@ -84,6 +84,28 @@ def transform(self, value):
             raise ValueError('Transform needs to be a callable.')
         self._transform = value
 
+    @property
+    def description(self):
+        return self._description
+
+    def set_description(self, description, overwrite=False):
+        """Update (add or overwrite) the dataset description.
+
+        Parameters
```

Comment:
overwrite is not present in parameter docstring

Reply:
Another option is not to have overwrite and use property setters and always overwrite... like https://github.com/braindecode/braindecode/blob/765cd046a63f051a96a9019a5322473b4a501129/braindecode/datasets/base.py#L77-L81 has property setters... but maybe this version safer?

Reply:
in this version if you do 
```python
dataset.description = whatever
```
it will just throw an error? or what happens?

Reply:
Well, do we want to always overwrite?

Reply:
Gives `AttributeError: can't set attribute`.

Reply:
I don't think we want to allow dataset.description = whatever, since it will destroy everything. The description should always be a pandas.Series / pandas.DataFrame. Also, my overwrite refers to individual columns in the Series/DataFrame that should be erplaced, not the entire object.